frontend/src/App.tsx

```diff
@@ -44,22 +59,51 @@ import {
   styled,
   DialogActions,
   LinearProgress,
+  List,
+  ListItem,
+  ListItemButton,
+  ListItemIcon,
+  ListItemText,
 } from '@mui/material'
 import MuiAlert from '@mui/material/Alert'
-import ContentCopyIcon from '@mui/icons-material/ContentCopy';
+import ContentCopyIcon from '@mui/icons-material/ContentCopy'
 import { api, handler } from '../wailsjs/go/models'
-import { EventsEmit, EventsOn } from '../wailsjs/runtime'
+import { BrowserOpenURL, EventsEmit, EventsOn } from '../wailsjs/runtime'
 import { Folder } from '@mui/icons-material'
 import CloseIcon from '@mui/icons-material/Close'
 import { BuildTime, Version } from '../wailsjs/go/main/about'
-import PodMountedInfo = handler.PodMountedInfo;
+import PodMountedInfo = handler.PodMountedInfo
+
+interface UserInfo {
+  username: string | any
+  password: string | any
+}
+
+interface AccountInfo {
+  userInfo: UserInfo[] | any
+  pods: PodMountedInfo[] | any
+}
+
+function createUserInfo(username: string, password: string): UserInfo {
+  return { username, password }
+}
+function addAccount(userInfo: UserInfo, pods: PodMountedInfo[]): AccountInfo {
+  return { userInfo, pods }
+}
+function createAccountInfo(
+  username: string,
+  password: string,
+  pods: PodMountedInfo[],
+): AccountInfo {
+  return { userInfo: createUserInfo(username, password), pods }
+}
```

Comment:
this is auto generated by wails from go in "frontend/wailsjs/go/models.ts". we should not move it in here. If anything changes on the go side we will not be able to use.